cmd/app.go

```diff
@@ -41,6 +42,7 @@ func init() {
 	flag.BoolVar(&enableLBPortMapping, "enable-lb-port-mapping", false, "enable port-mapping on the load balancer ports")
 	flag.StringVar(&gatewayChannel, "gateway-channel", "standard", "define the gateway API release channel to be used (standard, experimental, disabled), by default is standard")
 	flag.BoolVar(&enableDefaultIngress, "enable-default-ingress", true, "enable default ingress for the cloud provider kind ingress")
+	flag.StringVar(&proxyImage, "proxy-image", "docker.io/envoyproxy/envoy:v1.33.2", "proxy image to use for load balancers")
```

Comment:
lets' make this `docker.io/envoyproxy/envoy:v1.33.2` a constant `defaultProxyImage` or similar